contracts/interfaces/IWETHGateway.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
keep those functions  which we are going to use. that would help us to save gas while deploying  sc